packages/web-components/src/components/va-telephone/test/va-telephone.spec.tsx

```diff
@@ -32,52 +33,90 @@ describe('formatPhoneNumber', () => {
   });
 
   it('formats a TTY number', () => {
-    expect(VaTelephone.formatPhoneNumber(N11, null, null, null, true)).toBe('TTY: 911');
+    expect(VaTelephone.formatPhoneNumber(N11, null, null, null, true)).toBe('911');
+  });
+
+  it('formats an SMS number', () => {
+    expect(VaTelephone.formatPhoneNumber(sms, null, null, null, null, true)).toBe('123456');
   });
 });
 
 describe('formatTelLabel', () => {
   const contact = '8885551234';
   const extension = 123;
+  const smsContact = '123456';
   it('formats a phone number into an assistive tech label', () => {
-    expect(VaTelephone.formatTelLabel(contact, null, false, false)).toBe(
+    expect(VaTelephone.formatTelLabel(contact, null, false, false, false)).toBe(
       '8 8 8. 5 5 5. 1 2 3 4',
     );
   });
 
   it('formats a phone number with extension into an assistive tech label', () => {
-    expect(VaTelephone.formatTelLabel(contact, extension, false, false)).toBe(
+    expect(VaTelephone.formatTelLabel(contact, extension, false, false, false)).toBe(
       '8 8 8. 5 5 5. 1 2 3 4. extension. 1 2 3',
     );
   });
 
   it('formats a TTY phone number', () => {
-    expect(VaTelephone.formatTelLabel(contact, null, true, false)).toBe(
+    expect(VaTelephone.formatTelLabel(contact, null, true, false, false)).toBe(
       'TTY. 8 8 8. 5 5 5. 1 2 3 4',
     );
   });
 
   it('formats an international phone number', () => {
-    expect(VaTelephone.formatTelLabel(contact, null, false, true)).toBe(
+    expect(VaTelephone.formatTelLabel(contact, null, false, true, false)).toBe(
       '1. 8 8 8. 5 5 5. 1 2 3 4',
     );
   });
 
+  it('formats an SMS phone number', () => {
+    expect(VaTelephone.formatTelLabel(smsContact, null, false, false, true)).toBe(
+      'Text. 1 2 3 4 5 6',
+    );
+  });
+
+  it('creates an SMS phone number label', () => {
+    expect(VaTelephone.createTelLabel(false, true)).toBe(
+      'Text: ',
+    );
+  });
+
+  it('creates an TTY phone number label', () => {
+    expect(VaTelephone.createTelLabel(true, false)).toBe(
+      'TTY: ',
+    );
+  });
+
+  it('does not create a phone number label if SMS and TTY are false', () => {
+    expect(VaTelephone.createTelLabel(false, false)).toBe(
+      '',
+    );
+  });
+
+  it('does not create a phone number label if SMS and TTY are true', () => {
+    expect(VaTelephone.createTelLabel(true, true)).toBe(
+      '',
+    );
```

Comment:
This is a bit confusing to me, because reading the first test made me think that the next test would be that it _does_ create a phone number label if both of those args are true. I now see that it's about mutual exclusivity - the label is only created if one of the values is true, but not both. Can you make a change to help clear this up? I think even just adding a comment would work.

Reply:
After reverting the TTY changes, these tests are no longer relevant and have been removed.